### DISCUSSION TITLE
Lost all alerts from monitor interface

### DISCUSSION THREAD

**Discussion by dschmitz05** [Security-Onion-Solutions/securityonion#9769] [Unsupported Versions]
I am still getting my Sysmon alerts and playbook alerts, but for some reason, last night at 2200 my monitor interface stopped putting alerts out. So, no more Suricata alerts or anything else that is sent in on the monitor interface. The weird thing is that when I go do a tcpdump on the bond0, I can see all sorts of traffic.  I just updated to the latest version today and that did not fix it either. It worked for about 20 minutes and then all of a sudden quit again. Any ideas? I really do not want to do a reinstall because then I will have to completely retune and all of that not-so-fun stuff.

**Comment by InfosecGoon** [comment]
This is on a standalone installation?

Everything looks okay when you run so-status?

Are you seeing any logs written in /nsm/suricata or /nsm/zeek/logs/current?

**Reply by dschmitz05** [reply]
I do have logs in /nsm/suricata, however after I restarted my server, they stop coming in after 15 minutes. That is what is confusing me. When it first comes back online I had 5000 consumption EPS but now I am at 4.

**Reply by dschmitz05** [reply]
Everything does look good in so-status. What logs am I supposed to look at in /nsm/zeek/logs/current/ ? I do have logs in there, but I am unsure as to what you want me to look at.
